app/src/main/java/com/github/se/cyrcle/ui/profile/ViewProfileScreen.kt

```diff
@@ -253,7 +240,8 @@ private fun ProfileImage(
               contentAlignment = Alignment.Center) {
                 Icon(
                     imageVector = Icons.Filled.Person,
-                    contentDescription = "Default Profile Picture",
+                    contentDescription =
+                        stringResource(R.string.view_profile_screen_default_profile_picture),
```

Comment:
`contentDescription` are not displayed Strings, so no need to use the `strings.xml` offload

This comment is also valid for the line `256`, `270`, `332`

This comment is not blocking